src/main/java/com/training/assignments/Assignment.java

```diff
@@ -0,0 +1,69 @@
+package com.training.assignments;
+
+public class Assignment {
+
+    public double getDoubleFromInt(int number){
+        double doubleNumber=number;
+        return doubleNumber;
+    }
+
+    public int getIntFromDouble(double doubleNumber){
+        int intNumber =(int)doubleNumber;
+        return intNumber;
+    }
+
+    public float getFloatFromDouble(double doubleNumber){
+        float floatNumber =(float)doubleNumber;
+        return floatNumber;
+    }
+
+    public double getDoubleFromFloat(float floatNumber){
+        double doubleNumber =floatNumber;
+        return doubleNumber;
+    }
+
+    public byte getByteFromLong(long longNumber){
+        byte byteNumber =(byte)longNumber;
+        return byteNumber;
+    }
+
+    public long getLongFromByte(byte byteNumber){
+        long longNumber =byteNumber;
+        return longNumber;
+    }
+
+    public char[] getCharacterFromString(String name){
+        int index;
+        char[] characterArray=name.toCharArray();
+        char[] characters = new char[name.length()];
+        for(index=0;index<characterArray.length;index++){
+            characters[index] = characterArray[index];
+        }
+        return characters;
```

Comment:
1) char[] characterArray=name.toCharArray(); = When you do this you get string in character and its getting stored in characterArray varible, then you can directy return characterArray, No need of for loop